packages/back-end/src/services/stats.ts

```diff
@@ -67,6 +67,13 @@ export interface AnalysisSettingsForStatsEngine {
   max_dimensions: number;
 }
 
+export interface BanditSettingsForStatsEngine {
+  var_names: string[];
+  var_ids: string[];
+  decision_metric: string;
+  bandit_weights_seed: number;
```

Comment:
will be set as a random number/hash on every "create analysis and update weights" event, will be stored in the snapshot